dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpResponseMessageImpl.java

```diff
@@ -711,6 +712,36 @@ public void headerComplianceCheck() throws MessageSentException {
             updateCacheControl();
         }
 
+        if (getServiceContext().getHttpConfig().useHeadersConfiguration()) {
+            //Add all headers configured through the ADD configuration option
+            for (List<Map.Entry<String, String>> headers : getServiceContext().getHttpConfig().getConfiguredHeadersToAppend().values()) {
```

Comment:
Any reason you use `append` here rather than `add`?

Reply:
The idea was to match the `appendHeader()` API, but I have no objection for it to be inline with the configuration `add` format.